charts/csm-authorization/Chart.yaml

```diff
@@ -1,15 +1,15 @@
 apiVersion: v2
 name: csm-authorization
-version: 1.10.0
-appVersion: 1.10.0
+version: 1.10.1
+appVersion: 1.10.1
 type: application
 description: |
   CSM for Authorization is part of the [Container Storage Modules](https://github.com/dell/csm) open source suite of Kubernetes 
   storage enablers for Dell EMC storage products.  CSM for Authorization provides storage and Kubernetes administrators the ability
   to apply RBAC for Dell CSI Drivers.
 dependencies:
 - name: cert-manager
-  version: 1.11.0
```

Comment:
Why is this dependency being rolled back to an older version?

Reply:
Discussed over the call . Will be reverted back in CSM 1.11